korman/exporter/etlight.py

```diff
@@ -231,15 +233,24 @@ def _harvest_bakable_objects(self, objs):
             mods = i.plasma_modifiers
             lightmap_mod = mods.lightmap
             if lightmap_mod.enabled:
+                if lightmap_mod.bake_pass_name:
+                    bake_pass = bake_passes.get(lightmap_mod.bake_pass_name, None)
+                    if bake_pass is None:
+                        raise ExportError("Bake Lighting '{}': Could not find pass '{}'".format(i.name, lightmap_mod.bake_pass_name))
+                    lm_layers = tuple(bake_pass.render_layers)
```

Comment:
It would be nice to save the current layer visibility and restore them at the end.  Otherwise, we potentially enable a lot of layers (by default) that weren't enabled previously.